### PR TITLE
Make preliminary UX changes.

### PR DESCRIPTION
**Pull request by rjacobs31**

### CODE REVIEW

File: oneplus/misc_views.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from __future__ import division
+from __future__ import division
```
Comment (msinovcic) [inline oneplus/misc_views.py]: double import?

File: oneplus/misc_views.py [hunk 2]
```diff
@@ -118,7 +127,7 @@ def get():
         state['fname'] = ""
         state['sname'] = ""
         state['comment'] = ""
-        state['contact'] = ""
+        state['contact'] = "" if not usr or not usr.mobile else usr.mobile
```
Comment (msinovcic) [inline oneplus/misc_views.py:129]: erm?

**Review by msinovcic — CHANGES_REQUESTED** [review]

File: oneplus/misc_views.py (hunk 1 above)
Comment (rjacobs31) [reply oneplus/misc_views.py]: Whoops. Thanks for catching.

File: oneplus/misc_views.py (hunk 2 above)
Comment (rjacobs31) [reply oneplus/misc_views.py:129]: That's similar to a ternary operator in Java. This variable isn't allowed to be `None`, so the code is just replacing it with an empty string if the object or the field are `None`.

**Comment by msinovcic** [comment]
:+1:

**Pull request merged**